### PR TITLE
Add name of sections so correct main nav is active

### PR DESCRIPTION
**Pull request by Sgtpluck**
This addresses #198.

Adding an explicit name to the primary nav made it possible to create an "active"[ conditional based on the subnav name. definitely open to other ways of doing this!

i also cleaned up some orphaned pages, and added the sidebar subnav to some topics that were linked to but weren't directly on a page. 

i also got rid of a reference to the cloud.gov postmortem repo which either no longer exists or not everyone has access to it.

### CODE REVIEW

File: _guide/_data/navigation.yml [hunk 1]
```diff
@@ -70,43 +79,6 @@ languages:
   - text: CSS
     href: /css/
 
-# TODO: figure out how to nest links
```
Comment (Sgtpluck) [inline _guide/_data/navigation.yml:73]: it seemed like these weren't doing anything anymore -- is that accurate?

File: _guide/_pages/people/2016-Assessment-Guide.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
 ---
 title: 18F Engineering 2016 End of Year Assessment Guide
```
Comment (Sgtpluck) [inline _guide/_pages/people/2016-Assessment-Guide.md:2]: do we need both of these assessments? should they be updated? has the system changed?

File: _includes/components/header.html [hunk 3]
```diff
@@ -83,9 +83,7 @@
             </div>
             {% endif %}
             {% else %}
-              {% assign basedir = page.url | remove_first: '/' | split: '/' | first | lstrip %}
-              {% assign linkdir = _section.href |  replace: '/', ''  | lstrip %}
-              <a class="{% if _section.class %}{{ _section.class }}{% else %} usa-nav__link {% endif %} {% if linkdir == basedir %} usa-current{% endif %}" href="{% if _section.external %}{{ _section.href }}{% else %}{{ _section.href | relative_url }}{% endif %}">
+              <a class="{% if _section.class %}{{ _section.class }}{% else %} usa-nav__link {% endif %} {% if page.sidenav == _section.name %} usa-current{% endif %}" href="{% if _section.external %}{{ _section.href }}{% else %}{{ _section.href | relative_url }}{% endif %}">
```
Comment (Sgtpluck) [inline _includes/components/header.html:86]: this is the actual change that allows the top-level navigation to be active correctly.

**Comment by ryanhofdotgov** [comment]
@Sgtpluck I'll have some time to take a look at this in detail later this week, but saw that some of the changes were to the "protected" files for the [upstream template](https://github.com/18F/isildurs-bane), so those would be the natural place for those, if not in uswds-jekyll directly as appropriate. Basically anything outside of `_guide` will fall in that category.

I'm planning to retire that meta-template (it didn't get any uptake outside of this guide) but until then, those types of changes would be best made there until this guide gets un-entangled from that template.

IIRC, the header in the isildurs-bane template derived from the UX-guide which forked from uswds-jekyll, so it's been a long term goal to fold those changes back into uswds-jekyll. Here's an example of that: https://github.com/18F/isildurs-bane/issues/18

**Comment by Sgtpluck** [comment]
@ryanhofdotgov i can make the change upstream; it won't solve the isildurs-bane issue directly though, since the app-specific subnav names will have to be added to each individual repo using it, but i guess it won't be much *more* broken, and i can add some documentation to that effect. (Eng hiring was using it, but i remember there was that issue with the daily sync so i don't know if that's a concern.)

if i change it at the `uswds-jekyll` level that means it'll have to be an OR operation (rather than replacing the current conditional) since i wouldn't want to break sites that are currently working by forcing them to add a `name` key. (which is probably fine, just will look a bit messier)

thoughts?

File: _includes/components/header.html (hunk 3 above)
Comment (ryanhofdotgov) [reply _includes/components/header.html:86]: I'll add comments to https://github.com/18F/uswds-jekyll/pull/229

**Review by ryanhofdotgov — APPROVED** [review]
Cool! I'd lean towards ripping out the header changes in this PR and resolving upstream in uswds-jekyll, but I don't feel strongly enough about that.

It can just be transition to converging back to what usdws-jekyll provides, ultimately abandoning this component fork.

**Comment by ryanhofdotgov** [comment]
🤔  I may have to address #239 before any changes outside of `_guide` will stick. IIRC, the GitHub action will just restore template "owned" files on a daily basis.

**Comment by Sgtpluck** [comment]
thanks @ryanhofdotgov -- i will hold off on this change until your changes go through, it'll require a rebase. 😬

File: _guide/_pages/people/2017-Assessment-Guide.md [hunk 4]
```diff
@@ -25,7 +27,7 @@ In September 2016, a group of Engineering leadership and engineers met to determ
...
 
 ### Fair rating
-Individuals deserve fair ratings. These ratings should be calibrated both within Engineering and across 18F. The ratings and feedback should be aligned with the individual’s performance profile (as is required of course), and not biased towards individuals that the raters have interacted with more than others or those whose strengths align the most with their raters’ interests. Positive and constructive feedback should be balanced, and all feedback should be backed by evidence. Ideally the individual agrees with the rating they’re given.
+Individuals deserve fair ratings. These ratings should be calibrated both within Engineering and across 18F. The ratings and feedback should be aligned with the individual’s performance profile (as is required of course), and not biased towards individuals that the raters have interacted with more than others or those whose strengths align the most with their raters’ interests. Positive and construective feedback should be balanced, and all feedback should be backed by evidence. Ideally the individual agrees with the rating they’re given.
```
[the remark this answers is not in the document]
Comment (Sgtpluck) [reply _guide/_pages/people/2017-Assessment-Guide.md:30]: wow how did this even happen! yes thank you, great catch

File: _guide/_pages/people/2016-Assessment-Guide.md (hunk 2 above)
Comment (Sgtpluck) [reply _guide/_pages/people/2016-Assessment-Guide.md:2]: looking through this, it actually has a bunch of good information so i am going to generalize the 2017 one and take out dates but leave it in.

**Pull request merged**